src/inference/include/ie/ie_blob.h

```diff
@@ -806,7 +808,8 @@ class TBlob : public MemoryBlob {
     }
 };
 
-#ifdef __clang__
+// These should not be exported for WIN32 to avoid usage of '_handle' and '_allocator' across CRT bounaries
+#ifndef WIN32
```

Comment:
Is that for all compilers on Windows or should be replased with MSVC conpiler? `#ifndef _MSC_VER`

Reply:
Per my understanding, for Windows DLLs it is a general rule to avoid passing objects across CRT boundaries. If we really need these exports for 'clang+gcc' on linux to make binaries compatible, in this case at least for windows we can avoid using this.
Generally I don't like these exports at all, but these are here to fix some issues on Android. In this case my suggestion is to not use them at least for windows (it will avoid a huge number of warnings, 2 warnings for each instantiations for each client's cpp  file which includes ie_blob.hpp)

Reply:
should not we use `_WIN32` ? See https://stackoverflow.com/questions/662084/whats-the-difference-between-the-win32-and-win32-defines-in-c

Reply:
?